libres/lib/analysis/update.cpp

```diff
@@ -354,7 +382,8 @@ void run_analysis_update_with_rowscaling(
 
     ert::utils::Benchmark benchmark(logger,
                                     "run_analysis_update_with_rowscaling");
-    assert(parameters.size() > 0);
+    if (parameters.size() == 0)
+        return;
```

Comment:
Should this throw an error or warning of sorts?

Reply:
I would say no - this is a totally legitimate situation; one could move to the check to calling scope and if `parameters.size() == 0` not call the function at all.

Reply:
If we move the check to the calling scope, would it then not make sense to throw an error if `run_analysis_update_with_rowscaling` is called with parameter.size() == 0?

Reply:
No - the early return here is just a convenience, it is perfectly legitimate to run through this function with `paramaters = []` it is just a minor waste.

Reply:
Alright, I don't like it but can definitely live with it.